### DISCUSSION TITLE
!!! Handlungsbedarf !!! SENEC aktualisiert Firmware der Speicher

### DISCUSSION THREAD

**Discussion by ledermann** [solectrus/solectrus#2310] [Updates]
Ab heute, dem 29.08.2023, installiert SENEC eine neue Firmware auf den Speichern. Die SENEC-App informiert darüber mit diesem Hinweis:

![senec](https://github.com/solectrus/solectrus/assets/3557/8d5265e4-e936-4790-92d2-68b6e5b7e091)

# Wurde dein Speicher bereits aktualisiert?

Ob der eigene Speicher bereits aktualisiert worden ist, erkennst du wie folgt:

- Der Zugriff per `http` funktioniert **nicht** mehr, man muss `https` verwenden. Das lässt sich leicht im Browser ausprobieren, indem du `http://[ip-adresse]` bzw. `https://[ip-adresse]` in der Adresszeile eingibt. Die alte Firmware antwortet nur auf `http`, die neue hingegen nur auf `https`.
- Die neue Firmware nennt sich `Revision MCU 3825`, möglicherweise gibt es aber auch andere Versionsnummer. Die `25` am Ende scheint relevant zu sein.

# Was du nun tun musst

Damit SOLECTRUS weiterhin funktioniert, **muss zwingend** ein Update des SENEC-Collectors ([Version 0.9.0](https://github.com/solectrus/senec-collector/releases/tag/v0.9.0)) installiert werden und außerdem der Zugriff von `http` auf `https` umgestellt werden. In einem Blog-Posting habe ich das bereits beschrieben:
https://solectrus.de/blog/2023-08-18-unterstuetzung-fuer-neue-senec-firmware/

Es ist zu empfehlen, das Update bereits jetzt zu installieren, auch wenn der eigene Speicher noch mit der alten Firmware läuft. Das geht wie folgt, hier am Beispiel eines Raspberry Pi:

```
ssh pi@[RASPI-IP]
cd solectrus

docker compose pull
docker compose up -d
```

(Bei älteren Docker-Versionen wie z.B. auf Synology muss es `docker-compose` heißen, also mit Bindestrich)

Mit diesen Befehlen werden alle Container auf den aktuellen Stand gebracht, nicht nur der SENEC-Collector. Sämtliche Komponenten von SOLECTRUS funktionieren dann weiter, auf den Speicher wird unverändert per `http` zugegriffen.

Sobald das Firmware-Update auf deinem Speicher angekommen ist, wird dieser `http`-Zugriff nicht mehr funktionieren, d.h. SOLECTRUS wird als Status "Fail" anzeigen. Dies ist der Moment, wo auf `https` umgestellt werden muss. Dies geht wie folgt:

1. Editiere die Datei `.env` und ergänze eine **zusätzliche** Zeile mit `SENEC_SCHEMA=https` 
2. Editiere die Datei `docker-compose.yml` und ergänze unter `services.senec-collector.environment` den zusätzlichen Eintrag `SENEC_SCHEMA` (damit die Variable aus der `.env`-Datei auch im Container ankommt)
3. Starte die Container neu (`docker compose down`, dann `docker compose up -d`)

Dadurch greift der SENEC-Collector per `https` auf den Speicher zu und SOLECTRUS funktioniert wieder wie gewohnt.

**Comment by ChrisTheLink** [comment]
Hi,

was ein Zufall, eben wurde mein EnFluri getauscht und dabei fiel genau das Problem auf. Ich habe folgenden MCU Nummern:
**REVISION MCU: 0825**
REVISION MCU-BL: 2307
REVISION NPU-REGS: 10
REVISION NPU-IMAGE: 2309
REVISION GUI: 0966
REVISION INV: 214
REVISION NSP: 204/204
Lösung oben hat auf Anhieb funktioniert :-)
DANKE FÜR DEINE MEGA ARBEIT - der nächste Kaffe ist unterwegs!
Christoph

**Comment by Zeppelin-PV** [comment]
seltsam, ihr habt alle soviel Parameter in der Firmware Anzeige
bei mir ist es nur eine Hand voll, ist das abhängig von Optionen die man gebucht hat ?
... noch ist bei mir die Firmware die, die mit http: läuft
![grafik](https://github.com/solectrus/solectrus/assets/126392805/e62fe458-baa6-45d8-b97d-f602389abbca)

**Comment by jacitty** [comment]
Hi.
Danke für die Anleitung. Ich habe zwar alles exakt so gemacht, aber bekomme trotzdem keine Daten vom senec. Keine Ahnung, warum das so ist. Neueste Version ist auf dem Senec installiert, ich komme nur noch per https drauf.

**Reply by ledermann** [reply]
Im Log des SENEC-Collectors dürften Hinweise auf die Ursache stehen. Bitte poste mal, was dieser Befehl ergibt:

```
docker logs --tail 20 solectrus-senec-collector-1
```

Außerdem prüfen, ob wirklich `.env` **und** `docker-compose.yml` angepasst wurden und die Container danach neu gestartet wurden.

**Comment by jacitty** [comment]
Hallo. 
Wow, das ging ja extrem schnell :-)

Hier, das steht im Log und wiederholt sich ständig (IP ausgexxxt):

> Getting state names from SENEC by parsing source code...
> Error getting data from SENEC at http://192.xxx.xxx.xxx: Failed to open TCP connection to 192.xxx.xxx.xxx:80 (Connection refused - connect(2) for "192.xxx.xxx.xxx" port 80)
 
Sowohl der Raspi als auch der senec sind im gleichen Netz, das funkltionierte bis vor ca. 4 Stunden noch alles. .env und .yml sind angepasst, Updates werden jede nacht mittels cron gezogen, hab es aber eben auch nochmal manuell aufgerufen. Warum er hier die tcp-Connection ablehnt, keine Ahnung.

**Reply by jacitty** [reply]
Ok, Schande über mich. Ich hab den Fehler gefunden, Ich hatte in der .env nur nach _SCHEMA gesucht und hier aus http ein https gemacht. Ich hätte aber mal richtig lesen sollen und einen neuen Eintrag "SENEC_SCHEMA=https" machen sollen und dafür den Eintrag für "INFLUX_SCHEMA=http" in Ruhe lassen sollen.

Wenn man mal in mRuhe drüber nachdenkt klappts auch. Sorry für die Mühe die ich gemacht habe.

**Reply by ledermann** [reply]
Oh ja, `INFLUX_SCHEMA` ist für den Zugriff auf InfluxDB da, das darf nicht verändert werden.

Freut mich, dass es nun klappt 👍

**Comment by Zeppelin-PV** [comment]
Hilfe, ich bekomme das nicht hin
bekomme den neuen Eintrag nicht auf Linie
somit kommt ein Fehler nach docker compose pull

![grafik](https://github.com/solectrus/solectrus/assets/126392805/1157b7c4-257c-4ea1-be5e-81f6e12828ea)

**Reply by ledermann** [reply]
😄😄😄 Ich gehe davon aus, du verwendest den Texteditor `nano`, daher: Achtmal <kbd>Cursor-Rechts</kbd>, dann zweimal <kbd>Backspace</kbd>, dann <kbd>Strg+X</kbd>, dann <kbd>Y</kbd>, dann <kbd>ENTER</kbd>  

Wenn's nicht klappt, am besten mal nach der Bedienung von `nano` googlen.

**Comment by Zeppelin-PV** [comment]
![grafik](https://github.com/solectrus/solectrus/assets/126392805/2e8311c9-55a9-4405-b20e-c592e01052ec)

![grafik](https://github.com/solectrus/solectrus/assets/126392805/26c20f00-dec5-4ee9-9f3c-8c6fd217d75f)

**Reply by ledermann** [reply]
Die Einrückung der Zeilen ist falsch, die Bindestriche müssen alle untereinander stehen. Solange das nicht der Fall ist, gilt die gesamte Datei als inkorrekt und Docker wird das abweisen.

Ich kann dir da leider nicht wirklich weiterhelfen, das Editieren der Datei musst du selber hinbekommen. Mit Leertaste, Cursortasten und Backspace sollte das zu schaffen sein.

**Comment by Zeppelin-PV** [comment]
... läuft, DANKE !!!

**Comment by OnkelMF** [comment]
Grad eben "Fail" also ist das Update bei mir wohl auch heute angekommen. Sobald ich Daheim bin schau ich mal ins Log und ändere gemäß Deiner Anleitung - ich bin gespannt und werde berichten :)

**Reply by OnkelMF** [reply]
irgendwas übersehe ich, ich bekomme keine Daten. .env und die yml sind angepasst (hab auch nachgesehen):

Log: 

Getting state names from SENEC by parsing source code...
Error getting data from SENEC at 192.168.178.151: Failed to open TCP connection                         to 192.168.178.151:80 (Connection refused - connect(2) for "192.168.178.151" por                        t 80)

Kannst Du damit was anfangen ?

https://192.168.178.151/ << da erscheint das "normale" SENEC Fenster

**Reply by OnkelMF** [reply]
Erledigt, ich habe docker compose pull übersehen - danach erst docker compose up -d und schon lüppt dat :)

**Comment by nk-henning** [comment]
Heute bliebt bei mir solectrus nun auch leer. Ich war noch auf der 0.9.0, wenn ich nicht irre. Den Speicher erreiche ich nicht per https und auch http machte mir Probleme. 

Finally hier meine Versionen der Firmware:
REVISION MCU: 0825
REVISION MCU-BL: 2307
REVISION NPU-REGS: 10
REVISION NPU-IMAGE: 2309
REVISION GUI: 0966

Ich habe auf v0.13.0 geupdatet und bekomme nun auch wieder Daten. Jedoch sind die /now-Daten kompletter Müll. Draussen schon fast Mondschein und die Erzeugung soll wie bei Mittagssonne sein. Beim Status sehe ich auch "Fail, letzte Messung vor 7 Stunden"

Daher meine Frage: auch wenn ich den Speicher noch nicht via https ansprechen kann (und die Revision noch nicht 3825 ist: muss ich die Ergänzungen in der .env schon vornehmen und http anstatt https eintragen?

Vielen Dank für die Hilfe.

**Reply by nk-henning** [reply]
Ich liefere hiermit auch direkt die Lösung: JA, auch wenn noch nicht auf https umgestellt wurde, müssen die Einträge ergänzt werden. Solectrus läuft wieder. <3

**Reply by ledermann** [reply]
Freut mich, dass es geklappt hat - auch wenn ich nicht ganz nachvollziehen kann, was da passiert ist. Wenn nämlich die Variable `SENEC_SCHEMA` fehlt, wird automatisch `http` verwendet.

**Reply by OnkelMF** [reply]
825 ist genau die problematische Firmware - die habe ich seit heute Mittag auch und musste Solectrus "umbauen"

Beim Status sehe ich auch "Fail, letzte Messung vor 7 Stunden" << da hast Du doch die Lösung, Solectrus hat seit 7 Stunden keine Daten mehr und Du siehst die "letzte" Messung bevor der Speicher über http nicht mehr erreichbar war - das war nittags, da hat die PV offensichtlich ordentlich produziert und das steht ohne Umbau an solectrus da bis zum Sankt Nimmerleinstag ;)

**Comment by matrix753** [comment]
Ich habe auch das Problem, dass ich keine Daten von der SENEC-Hardware bekomme. Im Log bekomme ich das hier:

`Error getting data from SENEC at https://192.168.178.71: Failed to open TCP connection to 192.168.178.71:443 (Connection refused - connect(2) for "192.168.178.71" port 443)`

Die IP ist korrekt, https wird wohl verwendet. Pingen kann ich die IP auch.

Was muss ich tun?

**Reply by OnkelMF** [reply]
klingt so wie mein Problem das ich erst hatte, hast Du vielleicht "docker compose pull" übersehen ? Danach erst "docker compose up -d"
das war mein Fehler und ich hatte die gleiche Fehlermeldung !

**Reply by matrix753** [reply]
Ich habe erst `docker compose down` gemacht und dann `docker compose pull`. Dann kommt dieser Error:
![image](https://github.com/solectrus/solectrus/assets/144257723/ce7ea1f7-c70d-4b86-8bc0-900b375fcdb8)

**Reply by OnkelMF** [reply]
Docker compose down hab ich überhaupt nicht genutzt nur eben "pull" vergessenn daher liefs bei mir erst nicht, danach lief es und läuft nach wie vor ;)

**Reply by matrix753** [reply]
Das `docker compose pull` habe ich jetzt zum Laufen gebracht. Das war ein Netzwerk-Fehler, weil ich die Software in einer VM laufen lasse und die war nicht sauber connected.
![image](https://github.com/solectrus/solectrus/assets/144257723/1b310d2e-3dd2-4bba-bcb7-79006e0baf90)
Trotzdem kommt aber im Log immer noch dieser Fehler:
`Error getting data from SENEC at https://192.168.178.71: Failed to open TCP connection to 192.168.178.71:443 (Connection refused - connect(2) for "192.168.178.71" port 443)`
Und es laufen keine Daten von der SENEC-Hardware in die Software.

**Reply by OnkelMF** [reply]
jau -env und die .yml haste gecheckt, dass da die Einträge korrekt sind ? Wenn JA, fehlt ja noch "docker compose up -d" das muss IMMER nach "pull" gemacht werden sonst bringts ja nix ;)

**Reply by ledermann** [reply]
@matrix753 Ich denke, es ist so: Du hast auf `https` umgestellt, was man ja in deinem Docker-Log sehen kann. Aber dein Speicher läuft noch auf `http`. Das habe ich auch erst heute gelernt, dass es wohl mehrere Updates gibt, die SENEC installiert. Man kann also bereits einige Firmware-Updates haben, aber der Speicher läuft dennoch auf `http`. Daher ist Folgendes zu tun:

1) In `.env` wieder `SENEC_SCHEMA=http` einstellen
2) docker compose down
3) docker compose up -d

Erst wenn der Speicher wirklich auf `https` läuft (prüfen per Browser), dann `SENEC_SCHEMA=https` einstellen (und neu starten)

**Reply by matrix753** [reply]
@ledermann Tatsache! Ich hab jetzt wieder zurückgestellt auf http und jetzt kommen die Daten wieder. Ich bin nur stutzig geworden, weil ich heute von 12 bis 16 Uhr keine Daten bekommen habe. Dann dachte ich mir, dass ich auch umgestellt wurde, ohne dass ich im Browser auf https gecheckt hätte. Danke für den Hinweis!

**Comment by ledermann** [comment]
Bei mir ist das Update mittlerweile auch angekommen. In mehreren Schritten, mit mehreren Reboots. Die Umstellung auf `https` erfolgt übrigens erst durch `REVISION NPU-IMAGE: 2309`, die anderen Updates (MCU und GUI) haben damit offenbar nichts zu tun.

Ein paar Beobachtungen habe ich im Blog mal aufgeschrieben:
https://solectrus.de/blog/2023-09-06-erfahrungen-mit-dem-senec-update/

**Comment by Zeppelin-PV** [comment]
ja, das habe ich auch festgestellt. Der Zeitverlauf zwischen den einzelnen Updates waren in etwa 6 Stunden.
Was aber auffällt ist, dass trotz Umstellung des Speicher auf https in .env und docker die Daten manchmal aussetzten.
Auf dem Bild ist auch die neue Ladeunterbrechung ca. 6min Batteriediagnose zu sehen.
Danach Keine Daten für 8min.
alles ein wenig seltsam, seit der neuen Firmware
![grafik](https://github.com/solectrus/solectrus/assets/126392805/adce9b45-5818-4b34-934e-eb907adb7086)

**Reply by OnkelMF** [reply]
Das Problem hatte ich bisher noch nicht (Mein Update, bei mir kam alles auf einmal, war am 04.09.2023 ab ca. 13 Uhr laut Log). Solectrus läuft seit dem Umbau bisher problemlos durch. Batteriediagnose scheint er bei mir immer nachts zu machen. Fiel mir vor dem Update mal auf, dass nachts auf einmal der Netzbezug hoch ging, ohne das ein zusätzlicher Verbrauchen anging (ich hab zufällig am Laptop gespielt und da ist Solectrus immer offen) - in der SENEC App und auf dem Speicher konnte ich dann sehen, dass er gerade eine Batteriediagnose machte. Hatte mich vorher schon immer gewundert warum trotz nicht leerem Akku nachts immer wieder mal Strom aus dem Netz gezogen wird (Heizung aus, keine Wärmepumpe, kein E-Auto, keine Klimaanlage etc.) obwohl ich nur den üblichen Grundverbrauch habe. Das erklärte dass dann ;)

**Reply by ledermann** [reply]
Bei der Installation von Updates lässt sich ein Reboot oft nicht vermeiden. Währenddessen kann der Server natürlich nicht antworten. Auch Diagnoseläufe nach dem Update, in denen es irgendwelche Einschränkungen gibt, finde ich nachvollziehbar. Bei mir tauchte gestern und heute mehrmals "NPU-Fehler" und "Batteriediagnose" auf.

Das könnte SENEC zwar theoretisch abfedern, müsste dann aber einiges redundant auslegen, was die Komplexität der Software deutlichen erhöhen würde. Das lohnt sich nicht, nur um ein paar Minuten Ausfallzeiten zu vermeiden. 

Zu Bedenken ist ja auch: Die Schnittstelle, die SOLECTRUS nutzt, um Messwerte abzusaugen, gibt es offiziell ja gar nicht, sie wurde vermutlich nur für interne Zwecke von SENEC entwickelt.

**Reply by OnkelMF** [reply]
Siehste, die NPU-Fehler hatte ich am 4.9. auch massig im Log stehen, glaube 1-2 Mal noch am 5.9.  - seit dem nicht mehr. Scheint also ein "Problem" zu sein dass "nur" kurz nach dem Update auftritt.

**Comment by Zeppelin-PV** [comment]
... das ist es ja, was Solectrus so einzigartig macht !!!
Denke auch, dass speziell im Konditionierungsmodus zusätzlich einiges über die Schnittstellen läuft und den Datenbezug einschränken.

**Reply by OnkelMF** [reply]
Das glaube ich eher nicht, weil beim letzten Konditionierungsbetrieb lief Solectrus uneingeschränkt durch. So wie aktuell bei mir auch dieses Mal (auf Holz klopf). 

Was ich an Solectrus liebe: ich konnte problemlos nachvollziehen wie lange der letzte Konditionierungsbetrieb lief. Da SENEC ja kommentarlos die "Kulanzzahlung" überweist, hatte ich keine Ahnung wie die das berechnet haben - anhand der Daten aus Solectrus konnte ich sehen: passt ganz genau ;)

**Comment by UweRoggow** [comment]
Hallo zusammen, 
ich habe die Änderungen wie beschrieben vorgenommen. Jedoch habe ich einen Fehler. 
Was habe ich falsch gemacht ? Letzte Messung ist vor 6 Tagen 

![grafik](https://github.com/solectrus/solectrus/assets/144605630/17737697-f9fd-4d29-8cd2-5e1bfe61779e)

Grüße aus Magdeburg

**Reply by ledermann** [reply]
Hier würde ich den SENEC-Collector mal kontrollieren, am besten in dessen Logs gucken. Dort wird irgendeine Fehlermeldung zu sehen sein:

```
docker logs solectrus-senec-collector-1
```

Bitte stelle sicher:

- Du verwendest die aktuelle Version des SENEC-Collectors ([v0.9.0](https://github.com/solectrus/senec-collector/releases/tag/v0.9.0))
- Du hast in .env den Eintrag `SENEC_SCHEMA` richtig gesetzt (`https` bzw. `http`, je nachdem wie dein Speicher erreichbar ist) und `docker-compose.yml` angepasst
- Du hast nach einer Änderung die Container neu gestartet

**Comment by UweRoggow** [comment]
Hallo Georg, 

ich nutze eine Synology mit dem eigenen Docker. 
Um die Einstellungen vorzuhmen nutze ich PuTTY
Solectrus 13 ist mit Watchtower aktuell.

Wo bzw, muss ich den SENEC-Collector kontrollieren ? Wie kann ich die Logs prüfen ? 

die .env habe ich ergänzt
![grafik](https://github.com/solectrus/solectrus/assets/144605630/838aab80-e9b5-4cbe-9604-b2f2d46e86a9)

die docker-compose ist auch ergänzt
![grafik](https://github.com/solectrus/solectrus/assets/144605630/aec6c69c-2059-4c16-a3e2-82be131af520)

bei docker.compose pull erscheint folgende Fehlermeldung
![grafik](https://github.com/solectrus/solectrus/assets/144605630/ac868a57-e65d-4c7c-82eb-29ffa95c5331)

**Reply by ledermann** [reply]
Die Fehlermeldung "Permission denied" beim `docker-compose pull` liegt wohl am fehlenden `sudo`. In der [Installation-Anleitung](https://github.com/solectrus/hosting/blob/main/guide/synology/README.md) wird erklärt, wie man Docker so einstellen kann, dass man ohne `sudo` auskommt. `docker pull` brauchst du aber vermutlich nicht, wenn die Aktualisierung der Docker-Images bereits durch Watchtower erfolgt ist.

Logs prüfen geht so, wie schon geschrieben:

```
docker logs solectrus-senec-collector-1
```

(ggfs. `sudo` davor und Container-Namen anpassen gemäß output von `docker ps`)

Die Logs einzelner Docker-Container lassen sich auf Synology alternativ auch über die GUI im integrierten Container-Manager kontrollieren.

**Comment by Semmy100** [comment]
Ich habe solectrus in einen docker installiert. Wie muss ich da vorgehen, um senec auf https umzustellen?
Wo finde ich die Datei .env?

**Reply by ledermann** [reply]
Steht eigentlich alles oben. Hauptsächlich `SENEC_SCHEMA=https` in der `.env`, die `docker-compose.yml` anpassen, Images aktualisieren (`docker compose up`) und Container neu starten. Die `.env` findet sich im gleichen Ordner wie die `docker-compose.yml`, wegen des führenden Punktes gilt sie als "hidden".

Wenn es konkrete Probleme gibt, am besten hier ein Issue anlegen:
https://github.com/solectrus/hosting/issues

**Comment by OnkelMF** [comment]
Hallo Georg,

am 19.9 wurde das NPU Image bei mir auf 2310 aktualisiert. Grad aus dem Urlaub zurück und die letzten Logs durchgeschaut:

2023-09-19 10:04:08 [I|NPU] NPU: request_update changed to 1
2023-09-19 10:04:08 [I|NPU] NPU: permit_update changed to 1
2023-09-19 10:04:08 [I|INV_LV] Disabled Batteries and reset DC measurements
2023-09-19 10:04:08 [I|Energy Management] State: Akku voll -> Softwareaktualisierung
2023-09-19 10:04:09 [I|NPU] Start sysupgrade
2023-09-19 10:04:10 [I|INV_LV] Energysource transferred: Battery: 0 PV: 1 Island: 0
2023-09-19 10:04:10 [I|INV_LV] DC measurements taken
2023-09-19 10:04:12 [I|INV_LV] Event Appeared: BDC_is_not_Running
2023-09-19 10:04:12 [I|INV_LV] Event Disappeared: Relais_K1_Batt_closed
2023-09-19 10:04:17 [I|INV_LV] Event Disappeared: MiniDumploadActive
2023-09-19 10:06:12 [E|ServerCommunicator] Sending HTTP GET message failed with error: initialization failed
2023-09-19 10:06:37 [E|System Update] Sending HTTP message server status failed with error: initialization failed
2023-09-19 10:06:45 [I|NPU] Correctly started
2023-09-19 10:06:46 [I|NPU] Communication established
2023-09-19 10:06:46 [I|NPU] NPU: Change backend server to 0
2023-09-19 10:06:53 [I|NPU] NPU: image version 2309-b5746b50
2023-09-19 10:06:53 [I|NPU] NPU: protocol version 10 (0x0619)
2023-09-19 10:06:53 [I|NPU] NPU: image version 2310-b5746b50
2023-09-19 10:06:54 [I|NPU] NPU: request_update changed to 0
2023-09-19 10:06:54 [I|NPU] NPU: permit_update changed to 0
2023-09-19 10:06:54 [I|INV_LV] Enabled Batteries and reset DC measurements
2023-09-19 10:06:54 [I|Energy Management] State: Softwareaktualisierung -> Akku voll
2023-09-19 10:06:55 [I|INV_LV] Energysource transferred: Battery: 1 PV: 1 Island: 0
2023-09-19 10:06:56 [I|INV_LV] DC measurements taken
2023-09-19 10:06:59 [I|INV_LV] Event Appeared: Relais_K1_Batt_closed
2023-09-19 10:07:01 [I|INV_LV] Event Disappeared: BDC_is_not_Running
2023-09-19 10:07:09 [E|System Update] Sending HTTP message server status failed with error: initialization failed
2023-09-19 10:07:13 [I|INV_LV] Event Appeared: MiniDumploadActive
2023-09-19 10:07:27 [I|NPU] NPU: image version 2310-06196b91
2023-09-19 10:11:01 [I|ServerCommunicator] Reconnected.
2023-09-19 10:29:42 [I|BAT Interface] Battery temp: 26.00 to 29.00 °C, case temp: 32.11 °C
2023-09-19 10:29:42 [I|PV Interface] Protocol: 0xa0, No external inverter. INV_LV:[3514.8W] INV_HV:[583.5W] Total production: 4098.3W current feed-in limit: 100.00 %
2023-09-19 10:36:19 [I|NPU] Switched off
2023-09-19 10:36:49 [I|Energy Management] State: Akku voll -> NPU-Fehler
2023-09-19 10:36:54 [I|NPU] Correctly started
2023-09-19 10:36:56 [I|NPU] Communication established
2023-09-19 10:36:57 [I|NET] IP address changed from 192.168.178.151 to unconnected (255.255.255.255).
2023-09-19 10:37:26 [I|Energy Management] State: NPU-Fehler -> Akku voll
2023-09-19 10:37:38 [I|NPU] NPU: Change backend server to 0
2023-09-19 10:37:42 [I|NPU] NPU: image version 2310-06196b91
2023-09-19 10:37:42 [I|NPU] NPU: protocol version 10 (0x0619)

Solectrus läuft weiterhin die fehlerhaften Links zu "log" udn "Display" gehen weiterhin nicht, habe aber nach dem Updte keine NPU-Fehler mehr - andere Veränderungen sind mir nicht aufgefallen.

**Reply by ledermann** [reply]
Ja, kam bei mir auch kürzlich an. Ohne erkennbaren Effekt. Die NPU-Fehler hatten vorher bereits (also mit 2309) aufgehört.

**Reply by OnkelMF** [reply]
Ich hatte jetzt wieder paar Tage die üblichen NPU-Fehler, aber seit gestern ist damit wieder Schluss - die kriegen das echt nicht hin bei SENEC :(
Also die neue Version hat zumindest nichts verändert was Log und Display angeht, vermutlich also eher Sicherheitsprobleme als Bugfixing war der Grund für das "schnelle" Update....
